src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -843,6 +844,29 @@ kds_del_scu_context(struct kds_sched *kds, struct kds_client *client,
 	if (submitted == completed)
 		goto skip;
 
+	if (kds->xgq_enable) {
+		int wait_ms;
```

Comment:
Unitialized variable.
```suggestion
		int wait_ms = 500;
```